### PR TITLE
[SMALLFIX] Exclude concurrent sleeping UFS tests in non-default profiles

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

**Comment by calvinjia** [comment]
@gpang Thanks for the fix, should this be done for the other tests which are using sleeping ufs?

File: tests/src/test/java/alluxio/master/file/ConcurrentFileSystemMasterCreateTest.java [hunk 1]
```diff
@@ -87,8 +86,10 @@ public static void afterClass() throws Exception {
   }
 
   @Before
-  public void before() {
+  public void before() throws Exception {
     mFileSystem = FileSystem.Factory.get();
+    // Mount the sleeping UFS, to always the sleeping UFS over the ufs cluster.
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/master/file/ConcurrentFileSystemMasterCreateTest.java]: nit: I think this comment is inaccurate - it's still possible to use the default UFS by accessing a path not under `/sleep/`

Comment (gpang) [inline tests/src/test/java/alluxio/master/file/ConcurrentFileSystemMasterCreateTest.java]: Sure. I updated the comment to specify that these tests will only use the sleeping ufs.

**Comment by gpang** [comment]
@calvinjia Yes, it can apply to the other concurrent tests. However, what should we do about the `root` test cases?

**Comment by calvinjia** [comment]
@gpang I think to fix the currently failing tests your change is fine. As a long term solution, we should make the sleeping ufs tests only run in default profile where we can set the root mount point to sleep. Running these tests in other profiles with a mounted sleeping ufs is redundant.

**Comment by gpang** [comment]
@calvinjia I updated the pom to exclude the sleeping ufs tests for non-default profiles.

**Comment by calvinjia** [comment]
@gpang LGTM, I like this solution a lot more. Is there any way to avoid the repetition?

**Comment by calvinjia** [comment]
Jenkins, test this please.

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
@calvinjia I'm not sure. I briefly looked into having a profile activate another profile, but that isn't supported. http://stackoverflow.com/questions/943411/can-i-make-one-maven-profile-activate-another

**Pull request merged**